library/globals.inc.php

```diff
@@ -1772,13 +1772,28 @@ function gblTimeZones()
     //
     'Notifications' => array(
 
+
         'patient_reminder_sender_name' => array(
             xl('Patient Reminder Sender Name'),
             'text',                           // data type
             '',                               // default
             xl('Name of the sender for patient reminders.')
         ),
 
+      'SMS_GATEWAY_RETURN_NUMBER' => array(
+          xl('SMS Gateway Return Number'),
+          'text',                           // data type
+          '',                               // default
+          xl('Require within the United States.  Must sign up for two-way messaging.')
```

Comment:
wow, I reread my comment above and not very clear. Basically, the translation pipelines has problems with more than 1 space such as '  ' within the constants. So, just need to change the 2 spaces after your first sentence here to 1 space.